### DISCUSSION TITLE
Export Not only open Radioss

### DISCUSSION THREAD

**Discussion by mathcae** [WelSimLLC/WelSim-Apps#8] [General]
1.   I hope the software can export not only Radioss ,for example Ansys  abbaqus  lsdyna.
2.   I want the model tree to be able to group materials

**Comment by lcheng9** [comment]
Thank you for your suggestions. 
We will work on Item 1. Could you elaborate on the group materials in Item 2?

**Comment by mathcae** [comment]
We can set many groups:Aolly,Plastic,Rubber .so,it's easy to be found,instead of the only method of searching.



| |
***@***.***
|
|
***@***.***
|




---- Replied Message ----
| From | ***@***.***> |
| Date | 03/06/2023 06:43 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [WelSimLLC/WelSim-Apps] Export Not only open Radioss (Discussion #8) |

Thank you for your suggestions.
We will work on Item 1. Could you elaborate on the group materials in Item 2?

**Comment by lcheng9** [comment]
@chens08 Thank you for the elaboration. We will try to fulfill this feature.
